src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -418,7 +420,7 @@ public void testDestroyCacheWithWrongFileNameLength() throws Exception {
     }
 
     @Test
-    public void testCacheFile() throws Exception {
+    public void testCacheFileWithoutExternalResources() throws Exception {
```

Comment:
Yes, I think that it might be called as testCheckWhichDoesNotImplementExternalResourceHolderInterface. Are you Ok with the name? Methods can be grouped closer.